tests/unit/template_labels_test.go

```diff
@@ -0,0 +1,66 @@
+package unit_test
+
+import (
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"path/filepath"
+	"strings"
+)
+
+var _ = Describe("Verify k8ssandra template labels", func() {
+
+	var (
+		defaultKubeCtlOptions = k8s.NewKubectlOptions("", "",
+			defaultTestNamespace)
+		k8ssandraChartPath string
+	)
+
+	BeforeEach(func() {
+		path, err := filepath.Abs(chartsPath)
+		Expect(err).To(BeNil())
+		k8ssandraChartPath = path
+	})
+
+	Context("by rendering it with options", func() {
+
+		It("using only default options", func() {
+
+			options := &helm.Options{
+				KubectlOptions: defaultKubeCtlOptions,
+				SetValues: map[string]string{
+					"stargate.enabled":                              "true",
+					"repair.reaper.enabled":                         "true",
+					"backupRestore.medusa.enabled":                  "true",
+					"ingress.traefik.enabled":                       "true",
+					"ingress.traefik.monitoring.grafana.enabled":    "true",
+					"ingress.traefik.monitoring.prometheus.enabled": "true",
+				},
+			}
+
+			// Verify required labels for ea. k8ssandra template
+			requiredLabels := GetK8ssandraRequiredLabels()
+			templates := GetK8ssandraTemplates(k8ssandraChartPath)
+			for _, template := range templates {
+
+				var k8ssandraTemplates map[string]interface{}
+				idx := strings.Index(template, "templates")
+
+				targetTemplate := []string{template[idx:]}
```

Comment:
Is there a reason to cast it inside []string{} ? Wouldn't targetTemplate := template[idx:] be enough? I assume the underlying data structure is not going to be altered.

Reply:
Convenience for call to RenderTemplateE() only, expecting []string for templates.  Next version will adjust to string[] when passing in values as this isn't reused anywhere.